scripts/inactive_study.py

```diff
@@ -64,14 +67,23 @@
                        "for an OpenMC model divided into a number of layers")
 ap = ArgumentParser(description=program_description)
 
-ap.add_argument('-i', dest='script_name', type=str,
-                help='Name of the OpenMC python script to run (without .py extension)')
 ap.add_argument('-n-threads', dest='n_threads', type=int,
                 default=multiprocessing.cpu_count(), help='Number of threads to run Cardinal with')
-ap.add_argument('-input', dest='input_file', type=str,
+ap.add_argument('-i', dest='script_name', type=str, required=True,
+                help='Name of the OpenMC python script to run (without .py extension)')
+ap.add_argument('-input', dest='input_file', type=str, required=True,
                 help='Name of the Cardinal input file to run')
-
+ap.add_argument('--method', dest = 'method', choices =['all','half','window'], required=True,
+                help = "The type of way to detect steady state. Options are all, half, or window")
+opts, rem_args = ap.parse_known_args()
+if(opts.method == "window"):
+    ap.add_argument('--window_length', dest = 'window_length',required = True, type = int,
```

Comment:
Don't add parameters inside of if-statements. If the user does `./inactive_study.py --help`, then they would never see that `window_length` is a valid parameter. Please instead leave it as a top-level parameter, but throw an error if the user does _not_ specify if when using the window detection method.